### ISSUE TITLE
[DOC]: dfp-models in training-tuning-scripts does not have an environment.yml or requirements.txt

### ISSUE THREAD

**Issue by nyck33** [nv-morpheus/Morpheus#1226]
### How would you describe the priority of this documentation request

medium

### Describe the future/missing documentation

https://github.com/nv-morpheus/Morpheus/tree/branch-23.11/models/training-tuning-scripts/dfp-models

The other folders like `fraud-detection-models` does have one as do the other folders 

And I look at the first cell:

```
import os
import glob

import dill
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
from sklearn.metrics import confusion_matrix
from sklearn.metrics import f1_score
import torch

from morpheus.models.dfencoder import AutoEncoder
from morpheus.utils.seed import manual_seed
``` 
I have the dev container launched in VSCode and tried to run that first cell with the default morpheus environnment that is activated but of course I frist get `module dill not found`.  Then doing `conda list` I don't see torch nor sklearn.  

Then looking down further in that `hammah-20211017.ipynb` , I see a cell

```
# uncomment to train on other "role-g" instead of "user123"

# TRAINING_DATA_PATH = "../datasets/training-data/hammah-role-g-training-data.csv"
# VAL_DATA_PATH = "../datasets/validation-data/hammah-role-g-validation-data.csv"
# OUTPUT_MODEL_NAME = "hammah-role-g-20211017.pkl
```

I've run the script that downloads all the models and examples but don't see any `role-g` files at all so please advise.  

### Where have you looked?

See above.  

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

**Comment by jarmak-nv** [comment]
Hi @nyck33!
  
  Thanks for submitting this issue - our team has been notified and we'll get back to you as soon as we can!
  In the mean time, feel free to add any relevant information to this issue.
